src/UserFunctions.java

```diff
@@ -70,18 +74,25 @@ public bool parseCommand(String input) {
                 break;
             default:
                 guessWord(command);
+                break;
         }
 
         return true;
     }
 
-    private void printCommands() {
+    private String getNewLineCharacter() {
         String newline = "\n";
         String osName = System.getProperty("os.name");
         // check the OS because Windows is stupid and uses two characters for a newline
         if (osName.startsWith("Windows"))
             newline = "\r\n";
```

Comment:
Unlikely that we'd need to add to this one, but better safe than sorry.

```suggestion
        if (osName.startsWith("Windows")) {
            newline = "\r\n";
        }
```